orchagent/fdborch.cpp

```diff
@@ -1138,6 +1138,37 @@ void FdbOrch::flushFDBEntries(sai_object_id_t bridge_port_oid,
         }
     }
 }
+void FdbOrch::flushFdbByVlan(const string &alias)
+{
+    sai_status_t status;
+    swss::Port vlan;
+    sai_attribute_t vlan_attr[2];
+
+    if (!m_portsOrch->getPort(alias, vlan))
+    {
+        return;
+    }
+    SWSS_LOG_NOTICE("Start: Flush by vlan %s vlan_oid 0x%" PRIx64 " ",
```

Comment:
Can this be changed to INFO as the caller(stporch) is anyway Logging at NOTICE level and similar to flushFDBEntries